src/p4est_mesh.c

```diff
@@ -162,6 +171,16 @@ mesh_edge_process_inter_tree_edges (p8est_iter_edge_info_t * info,
   equads = P4EST_ALLOC (p4est_locidx_t, nAdjacentQuads);
   eedges = P4EST_ALLOC (int8_t, nAdjacentQuads);
 
+  ccorners = NULL;
+  cquads = NULL;
+  if (mesh->params.edgehanging_corners == 1 &&
```

Comment:
Please check booleans with if (a) or if (!a). The C standard considers any nonzero as true.